contracts/mixnet/src/queries.rs

```diff
@@ -144,6 +144,24 @@ pub(crate) fn query_mixnode_delegations_paged(
     ))
 }
 
+pub(crate) fn query_all_mixnode_delegations_paged(
+    deps: Deps,
+    start_after: Option<Vec<u8>>,
+    limit: Option<u32>,
+) -> StdResult<PagedAllDelegationsResponse> {
+    let limit = limit
+        .unwrap_or(DELEGATION_PAGE_DEFAULT_LIMIT)
+        .min(DELEGATION_PAGE_MAX_LIMIT) as usize;
+
+    let bucket = all_mix_delegations_read::<RawDelegationData>(deps.storage);
+    let mut start = start_after.map(|mut v| {
+        v.push(0);
```

Comment:
why are you pushing the 0 byte here?

Reply:
Appending a 0 byte basically gets you to the next possible key, in the key byte array. We do the same thing in other places to obtain the `start_after` places: (i.e. https://github.com/nymtech/nym/blob/020cad897d208cb21d3cd5c73294f02ea2881d05/contracts/mixnet/src/storage.rs#L187)